tests/conftest.py

```diff
@@ -586,10 +586,9 @@ def ogm_incremental_harvester():
 
 @pytest.fixture
 def ogm_full_record_set():
+    """Full set of identifiers after suppressed and invalid records skipped."""
```

Comment:
[Non-blocking] In `tests/fixtures/ogm/files/edu.pluto`, I see that there are three XML files in the directory. How are there only two identifiers for `pluto:` in `ogm_full_record_set`? 🤔

Reply:
Good question, had to look it up myself, definitely not obvious.

The answer is that those files are used in [this fixture](https://github.com/MITLibraries/geo-harvester/blob/GDT-329-skip-ogm-supressed/tests/conftest.py#L706-L723) to create some file churn in a simulated git repository.  In that link, you can see that [record2.xml is actually removed](https://github.com/MITLibraries/geo-harvester/blob/GDT-329-skip-ogm-supressed/tests/conftest.py#L720) from this simulated git repository.  So we have 3 files to work with, but the fixture ultimately creates a temporary git repository that only has two files for the harvester to get.